src/test/objectstore/store_test.cc

```diff
@@ -3092,14 +3092,15 @@ TEST_P(StoreTest, LongnameSplitTest) {
     t.create_collection(cid, 0);
     cerr << "Creating collection " << cid << std::endl;
     r = apply_transaction(store, &osr, std::move(t));
-    ASSERT_EQ(r, 0);
+    ASSERT_EQ(0, r);
```

Comment:
the first param is the expected value, and the second is the actual value. hence this change.